/oak:
  ros__parameters:
    pipeline_gen:
      i_nn_type: none
      i_pipeline_type: Depth
    right:
      i_publish_topic: true
    stereo:
      i_extended_disp: true
      i_board_socket_id: 1
      i_enable_left_rgbd: true
